source/sync/get-started.txt

```diff
@@ -69,17 +73,17 @@ check out the SDK docs:
 
 .. note::
 
-   The Realm Web SDK does not currently support Realm Database or Atlas Device Sync.
-   However, you can query the same data in an Atlas cluster using 
+   The Web SDK does not currently support Realm Database or Atlas Device Sync.
```

Comment:
I don't think this is true anymore, and it never got updated. The Web SDK has a Device Sync Preview, and has had for some time: https://www.mongodb.com/docs/atlas/device-sdks/web/sync/

Reply:
thank you!